rules/coding-style/src/Rector/Throw_/AnnotateThrowablesRector.php

```diff
@@ -93,59 +117,77 @@ public function throwException(int $code)
         );
     }
 
-    /**
-     * @param Throw_ $node
-     */
     public function refactor(Node $node): ?Node
```

Comment:
There should be same nodes as in `getNodeTypes()` method now

Reply:
Done